app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes, through: :posts
+  has_many :comments, through: :posts
+
+  def most_recent_posts
+    posts.last(3)
+  end
+
+  def increment_posts_counter
+    user.posts_counter += 1
+  end
```

Comment:
I suggest we move this to the `post` model as a method that updates the posts counter for a user. What do you think? 🤔 

> ![image](https://user-images.githubusercontent.com/66112760/150020348-5cbb25da-32d6-4d7a-8215-bcfb3fbd4bda.png)